clang_tidy/__init__.py

```diff
@@ -6,15 +6,24 @@
 def _get_executable(name):
     return os.path.join(os.path.dirname(__file__), "data", "bin", name)
```

Comment:
This would make the test a little nicer to write
```suggestion
    from pathlib import Path
    return Path(__file__).resolve().parent / "data" / "bin" / name
```